### ISSUE TITLE
ArgoCD Applications List Tile Titles Prefixed with `argocd/`

### ISSUE THREAD

**Issue by booleanbetrayal** [argoproj/argo-cd#11173]
Checklist:

* [x] I've searched in the docs and FAQ for my answer: https://bit.ly/argocd-faq.
* [x] I've included steps to reproduce the bug.
* [x] I've pasted the output of `argocd version`.

**Describe the bug**

Every Application in the Applications > Tiles view has its title prefixed with "argocd/" for some reason, despite living in different namespaces / clusters. In the Application detail view, the title is rendered appropriately without this prefix. These Applications are not tied to ApplicationSets.

**To Reproduce**

Create a series of Applications and Projects via the argocd-apps Helm values conventions and view the site in ArgoCD v2.5.1.

**Expected behavior**

Applications should render the title based on the defined Application name, without any `argocd/` prefix.

**Screenshots**

Rendered Incorrectly:

<img width="1098" alt="Screen Shot 2022-11-02 at 5 29 46 PM" src="https://user-images.githubusercontent.com/981825/199622309-a3329c86-a79f-4732-8c88-7a9178c20ad3.png">

Rendered Correctly:

<img width="873" alt="Screen Shot 2022-11-02 at 5 40 50 PM" src="https://user-images.githubusercontent.com/981825/199622544-8b26f6b9-3b1c-4ac7-81e3-316e8f1f3787.png">

**Version**

```shell
argocd: v2.5.1+504da42
  BuildDate: 2022-11-01T21:36:58Z
  GitCommit: 504da424c2c9bb91d7fb2ebf3ae72162e7a5a5be
  GitTreeState: clean
  GoVersion: go1.18.7
  Compiler: gc
  Platform: linux/amd64
```

**Logs**

N/A

**Comment by Brice187** [comment]
Please paste the (sanitized) output of `kubectl get applications.argoproj.io -A`

**Comment by suraj2410** [comment]
there was a similar discussion https://github.com/argoproj/argo-cd/issues/11111 
but none the less some flag to disable this behavior of namespace grouping in the UI would be nice

**Comment by iam-veeramalla** [comment]
This seems to be introduced by apps in namespace feature, please see #11111

**Comment by booleanbetrayal** [comment]
Closing as a duplicate of #1111 which is anticipated to be fixed by #11165
